.gitignore

```diff
@@ -10,3 +10,4 @@ node_modules/*
 .projectile
 .tern-port
 TAGS
+.DS_Store
```

Comment:
I don't think OS-related files should be added in the .gitignore 🤔

Reply:
You're probably right, but maybe it can avoid peoples pushing wrong things ? I mean, it cost nothing :/

Reply:
It must be move to your global gitignore